GOVERNANCE.md

```diff
@@ -3,16 +3,31 @@
 This document describes the governance process under which the Serverless
 Working Group (WG) will manage this repository.
 
-## Working Group Meetings
+## Working Group Process
 
 In order to provide equitable rights to all Working Group members,
 the following process will be followed:
 
 * Official WG meetings will be announced at least a week in advance.
-* Proposed changes to any document will be done via a Pull Request (PR).
-* PRs will be reviewed during official WG meetings.
-* During meetings, priority will be given to PRs that appear to be ready for
-  a vote over those that appear to require discussions.
+* Proposed changes to any document will be done via a Pull Request (PR)
+* PRs will be reviewed in advance of WG meetings
```

Comment:
This feels a bit odd to say since we can't really enforce it.

Reply:
sure we can... right now our governance model is moving things forward by default when there isn't clear support from multiple members of the group

Reply:
@ultrasaurus that's true, however, mandating PRs are reviewed prior to the call can be difficult to enforce in terms of determining how many reviewers will have to have chimed-in on a PR in advance for it to be discussed and in terms of the day that we reach a nominal backlog (only have a couple PRs open to review). 

We can encourage pre-review and optimize our time spent on those first.

Reply:
Right now, PRs which have had little to no discussion are prioritized, which seems to be an unwritten rule which IMHO is not optimal.  Discussion is not always a sign of disagreement.